core/src/main/resources/hudson/Messages_pl.properties

```diff
@@ -0,0 +1,26 @@
+# The MIT License
+#
+# Copyright (c) 2013, Chunghwa Telecom Co., Ltd., Pei-Tang Huang
```

Comment:
Hi, please replace it with 2016 and your own name. Thanks
Le 17 janv. 2016 9:45 PM, "Damian Szczepanik" <notifications@github.com> a
écrit :

> In core/src/main/resources/hudson/Messages_pl.properties
> <https://github.com/jenkinsci/jenkins/pull/1986#discussion_r49952059>:
>
> > @@ -0,0 +1,26 @@
> > +# The MIT License
> > +#
> > +# Copyright (c) 2013, Chunghwa Telecom Co., Ltd., Pei-Tang Huang
>
> I've copied this from another properties file
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jenkinsci/jenkins/pull/1986/files#r49952059>.
>